.github/workflows/release.yml

```
@@ -173,7 +173,7 @@ jobs:
 
       - name: Tag and push latest docker image
         # Only tag and push `latest` when it's not a phylum-ci pre-release
-        if: ! github.event.inputs.prerelease
+        if: github.event.inputs.prerelease == false
```

I believe there is another option worth considering:

```suggestion
        if: ${{ !github.event.inputs.prerelease }}
```

The `${{ }}` is optional in an `if` clause so I usually recommend going without it, but including it will fix the YAML validation issues.

---

Testing the proposed option of `if: ${{ !github.event.inputs.prerelease }}` showed that it does not work as expected...it evaluates as false no matter what the value of `prerelease` is.

After further review, @kylewillmon discovered a note in the [workflow_dispatch documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs) that states:

> Note: The workflow will also receive the inputs in the
> `github.event.inputs` context. The information in the `inputs` context
> and `github.event.inputs` context is identical except that the
> `inputs` context preserves Boolean values as Booleans instead of
> converting them to strings.

Making the change to use `inputs` universally helps to simplify the understanding of how these inputs are provided and therefore how they can be used.